homeassistant/components/switch/rest.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""
+homeassistant.components.switch.rest
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Allows to configure a REST switch.
+
+Configuration:
+
+switch:
+  platform: rest
+  name: "Bedroom Switch"
+  resource: "http://IP_ADDRESS/ENDPOINT"
+  body_on: "ON"
+  body_off: "OFF"
```

Comment:
People using this are going to be more familiar with REST and HTTP terms than they are with the Requests python library. It should probably match the context of the component, not the underlying library.